src/main/java/org/apache/commons/csv/pretty/TableStyle.java

```diff
@@ -0,0 +1,220 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.commons.csv.pretty;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * This class packs all the information about the style of a table.
+ * We have prepared two pre-defined styles:
+ * <ul>
+ *     <li><code>SIMPLE</code> uses simple ASCII characters to draw the table</li>
+ *     <li><code>FANCY</code> uses so-called Extended ASCII characters to draw fancier tables</li>
+ * </ul>
+ *
+ *
+ * @author Ali Ghanbari
+ */
+public final class TableStyle implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    public static final TableStyle SIMPLE = new TableStyle('-',
+            '|',
+            '=',
+            '|',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+',
+            '+');
+
+    public static final TableStyle FANCY = new TableStyle('─',
+            '│',
+            '═',
+            '║',
+            '┬',
+            '╤',
+            '┴',
+            '╧',
+            '┼',
+            '╪',
+            '├',
+            '┤',
+            '╟',
+            '╢',
+            '╠',
+            '╣',
+            '┌',
+            '╔',
+            '┐',
+            '╗',
+            '└',
+            '╚',
+            '┘',
+            '╝');
+
+    public final char THIN_HORIZONTAL_LINE;
```

Comment:
Instance variables should be in lower-case and camel-cased, NOT in upper case, for example `THIN_VERTICAL_LINE` should be `thinVertialLine`. Also, public instance variables break encapsulation.